src/windows.rs

```diff
@@ -578,7 +586,7 @@ ffi! {
 
     pub const ARM_MAX_BREAKPOINTS: usize = 8;
     pub const ARM_MAX_WATCHPOINTS: usize = 1;
-
+    
```

Comment:
Looks like some stray whitespace here and above?

Reply:
Oops, will fix